demo_interfaces/Header header
string robot_config
string protocol_config

string robot_ip
bool simulate False

# string rc_path None
# string pc_path None